### PR TITLE
ratelimit support stat prefix

### PR DESCRIPTION
**Pull request by StarryVae**
Commit Message: ratelimit support stat prefix
Additional Description: N/A
Risk Level: low
Testing: ut
Docs Changes: N/A
Release Notes: N/A

### CODE REVIEW

**Comment by StarryVae** [comment]
issue #27421

File: api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto
Comment (htuch) [inline api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto:136]: Should update ratelimit stats docs as well: https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/rate_limit_filter.html#statistics

**Review by htuch — CHANGES_REQUESTED** [review]
LGTM modulo nit.

File: api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto
Comment (StarryVae) [reply api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto:136]: done, ptal again, thanks.

**Review by htuch — APPROVED** [review]
LGTM, thanks!

**Comment by StarryVae** [comment]
the ci seems to be blocked by flake coverage #30701

**Comment by htuch** [comment]
@StarryVae can you rebase and push again? Thanks.

**Comment by StarryVae** [comment]
done, thanks. @htuch

**Review by htuch — APPROVED** [review]

**Pull request merged**
